pkg/controller/util/util.go

```diff
@@ -58,8 +58,7 @@ func Addresses(ctx context.Context, client client.Client) ([]string, error) {
 		return nil, err
 	}
 
-	addresses := make([]string, len(nodeList.Items))
-
+	addresses := []string{}
```

Comment:
Why are you doing this?

Reply:
so basically when you initialize the addresses with one "empty" item, and then append to it we will end up with a slice with two items, one of them is empty, I just figured this should be easier